### PR TITLE
docs(usermanual): add documentation about new features

### LINKED ISSUE #3187: Add features documentation
This Issue not for solve, but for storage list of features, which need to add in user manual
- [x] Saved friend requests and groupchat invites (#3051)
- [x] Markdown (#3136)
- [x] Action messages `/me`

### PR DESCRIPTION
**Pull request by sudden6**
fixes #3187, #3136 

I will also commit a fix for https://github.com/tux3/qTox/issues/3051 later on.

### CODE REVIEW

File: doc/user_manual_en.md [hunk 1]
```diff
@@ -240,6 +241,10 @@ Additionally, qTox supports three modes of Markdown parsing:
 
 *Note that any change in Markdown preference will require a restart.*
 
+qTox also supports action messages by prefixing a message with `\me`, where
+`\me` is replaced with your current username. For example `\me likes cats`
```
Comment (zetok) [inline doc/user_manual_en.md]: `s,\\,/,g` ?

File: doc/user_manual_en.md [hunk 2]
```diff
@@ -240,6 +241,10 @@ Additionally, qTox supports three modes of Markdown parsing:
...
+turns into `qTox User likes cats`.
```
Comment (zetok) [inline doc/user_manual_en.md]: ~~would be `* qTox user ..`~~

Edit: actually: _`* qTox User ..`_

File: doc/user_manual_en.md (hunk 1 above)
Comment (SkyzohKey) [inline doc/user_manual_en.md]: Yeah, that's `/me` not `\me` x')

File: doc/user_manual_en.md [hunk 3]
```diff
@@ -240,6 +241,10 @@ Additionally, qTox supports three modes of Markdown parsing:
 
 *Note that any change in Markdown preference will require a restart.*
 
+qTox also supports action messages by prefixing a message with `/me`, where
+`/me` is replaced with your current username. For example `/me likes cats`
+turns into *`` * qTox User``*` likes cats`.
```
Comment (zetok) [inline doc/user_manual_en.md]: uh, sorry, what I wrote was kinda misleading – it was supposed to mean that whole sentence, not just user name turns italic

**Pull request merged**